setup.py

```diff
@@ -1,11 +1,27 @@
 #!/usr/bin/env python
 
+VERSION = '1.0.0'
+
 import glob
 from setuptools import setup
 setup(
     name = "hocr_tools",
-    version = "0.1",
-    author = 'Thomas Breuel',
+    version = VERSION,
     description = 'Advanced tools for hOCR integration',
+    author = 'Thomas Breuel',
+    author_email = 'tmbdev@gmail.com',
```

Comment:
It might also make sense to add here
```python
    maintainer = '',
    maintainer_email = '',
```
with the credentials of one of us. :innocent: Do you volunteer @kba?